### PR TITLE
docs(list): updated documentations

### PR DESCRIPTION
**Pull request by FlorentDinet**
# General summary

Added minimum heights, padding and clickable states info and demos

# Screenshots

Capture d’écran 2021-12-23 à 15 49 37

### CODE REVIEW

**Review by gcornut — APPROVED** [review]

File: packages/site-demo/content/product/components/list/index.mdx [hunk 1]
```diff
@@ -2,22 +2,54 @@
...
+
+`tiny`, `regular` and `big` list items have specific **minimum heights** and a `8px` vertical padding.
+Favor those 3 sizes for lists with list items with consistent heights.
+
+`huge` list items have a `16px` vertical padding.
+Use this size for any other cases.
+
+
+### Tiny
+Tiny list item have a minimum height of `36px`.
```
Comment (GregBillo) [inline packages/site-demo/content/product/components/list/index.mdx:17]: items

File: packages/site-demo/content/product/components/list/index.mdx [hunk 2]
```diff
@@ -2,22 +2,54 @@
...
+Use this size for dense lists with single ine items like menus made of [dropdown](../dropdown).
 
 <DemoBlock demo="tiny" />
 
-## Regular
+### Regular
+Regular list item have a minimum height of `52px`.
```
Comment (GregBillo) [inline packages/site-demo/content/product/components/list/index.mdx:23]: items

File: packages/site-demo/content/product/components/list/index.mdx [hunk 3]
```diff
@@ -2,22 +2,54 @@
...
+Use this size for single line list items.
 
 <DemoBlock demo="regular" />
 
-## Big
+### Big
+Big list item have a minimum height of `72px`.
```
Comment (GregBillo) [inline packages/site-demo/content/product/components/list/index.mdx:29]: items

File: packages/site-demo/content/product/components/list/index.mdx [hunk 4]
```diff
@@ -2,22 +2,54 @@
...
+Use this size for two lines list items.
 
 <DemoBlock demo="big" />
 
-## Huge
+### Huge
+Huge list item have a vertical padding of `16px`.
```
Comment (GregBillo) [inline packages/site-demo/content/product/components/list/index.mdx:35]: items

File: packages/site-demo/content/product/components/list/index.mdx [hunk 5]
```diff
@@ -2,22 +2,54 @@
...
+Use this size for list items that have more than two lines of content.
 
 <DemoBlock demo="huge" />
 
+## Item padding
+Horizontal padding has 2 sizes: `big` and `huge`.
```
Comment (GregBillo) [inline packages/site-demo/content/product/components/list/index.mdx:41]: The horizontal padding has

**Pull request merged**